src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/OutputFactory.java

```diff
@@ -220,6 +222,22 @@ public Void visitScoped(Declaration.Scoped d, Declaration parent) {
         return null;
     }
 
+    private static final boolean isVaList(FunctionDescriptor desc) {
```

Comment:
Ugh - too bad there's no way to detect this in libclang - not your fault of course!